src_c/mixer.c

```diff
@@ -100,8 +100,8 @@ struct ChannelData {
 static struct ChannelData *channeldata = NULL;
 static int numchanneldata = 0;
 
-Mix_Music **current_music;
-Mix_Music **queue_music;
+Mix_Music **mx_current_music;
```

Comment:
ps. We were trying to move everything to a pg_ prefix, but hadn't converted all the files yet.